static/js/popovers.js

```diff
@@ -815,10 +815,8 @@ export function register_click_handlers() {
     $("body").on("click", ".info_popover_actions .narrow_to_private_messages", (e) => {
         const user_id = elem_to_user_id($(e.target).parents("ul"));
         const email = people.get_by_user_id(user_id).email;
+        overlays.close_active();
```

Comment:
We should first check whether any overlay is active or not before calling `close_active`.

Reply:
Also, would be better to explain this change in the commit message.

Reply:
> Also, would be better to explain this change in the commit message.

Should I make a separate commit for this or just add the explanation in the original commit message?